helm/robusta/values.yaml

```diff
@@ -554,6 +554,17 @@ platformPlaybooks:
   sinks:
     - "robusta_ui_sink"
 
+- name: "RobustaMaintenance"
+  triggers:
+  - on_schedule:
+      cron_schedule_repeat:
+        cron_expression: "0 * * * *" # every hour
```

Comment:
I think we can run less than every hour
I think every few hours is probably enough

Reply:
okay, every 3 hours